dist/gcc-compatible/Hacl_Streaming_Functor.h

```diff
@@ -37,25 +37,17 @@ extern "C" {
 
 
 #include "evercrypt_targetconfig.h"
-typedef uint8_t EverCrypt_Helpers_uint8_t;
+typedef void *Hacl_Streaming_Functor_invariant_s;
```

Comment:
should we mark these definitions (`invariant_s`, `freeable`, `preserves_freeable`, `invariant`) as `inline_for_extraction noextract`?

Reply:
same reason, I added suitable flags except for those that were generating new files... this is now fixed and this file is gone, too